local-dev/.env-example

```diff
@@ -18,6 +18,14 @@ CARGO_TAG=develop
 CARGO_PORT_GRPC=50000
 CARGO_PORT_REST=8000
 
+# -----------------------------------------------------------
+# svc-assets
+# -----------------------------------------------------------
+CARGO_IMAGE=ghcr.io/arrow-air/svc-assets
+CARGO_TAG=develop
+CARGO_PORT_GRPC=50004
+CARGO_PORT_REST=8001
```

Comment:
To keep things less confusing, please use the same last number for the ports, so it's clear they're ending up at the same server.
In this case you can change the REST port to 8004